solidity/random-beacon/test/RandomBeacon.GroupCreation.test.ts

```diff
@@ -217,18 +210,6 @@ describe("RandomBeacon - Group Creation", () => {
                 "Not awaiting genesis"
               )
             })
-
-            it("should refund ETH", async () => {
```

Comment:
This test validates the refunded ETH after `.approveDkgResult(dkgResult)` call done in `before(..)`. In every `approveDkgResult` call we do refund ETH to the caller. `submitterInitialBalance` returned from `await signAndSubmitCorrectDkgResult()` is needed to calculate the difference between the initial caller's balance and what it has after the refund.

It is true that we have also tests in `approveDkgResult` but each refund scenario returns a slightly different result. This is how the offset was chosen by running different tests and picking a number that works for many scenarios.

Same comment for the other "should refund ETH" tests that were removed in this commit.

Reply:
Hmm... I would then improve test cases for `approveDkgResult` in case we need to cover more scenarios. If we place assertions for refunding ETH for functions that are not supposed to refund ETH, this is getting confusing.

Also, please see https://github.com/keep-network/keep-core/pull/2860/commits/701e0686698c306cd38929dc71f43ad384aefbbe.

Two tests are still failing (one added in the referenced commit), I think we may either need to adjust the offset or give up on the precision.